packages/models/src/purpose/protobufConverterFromV1.ts

```diff
@@ -79,16 +79,24 @@ export const fromPurposeRiskAnalysisFormV1 = (
   })),
 });
 
-export const fromPurposeV1 = (input: PurposeV1): Purpose => ({
-  ...input,
-  id: unsafeBrandId(input.id),
-  eserviceId: unsafeBrandId(input.eserviceId),
-  consumerId: unsafeBrandId(input.consumerId),
-  versions: input.versions.map(fromPurposeVersionV1),
-  isFreeOfCharge: input.isFreeOfCharge || true,
```

Comment:
Fix. This was always `true`. The desired behavior is that `true` is the fallback when `isFreeOfCharge` is `undefined`.

Reply:
Nice catch